common/internal_model/src/main/scala/weco/catalogue/internal_model/identifiers/IdentifierType.scala

```diff
@@ -125,4 +125,10 @@ object IdentifierType extends Enum[IdentifierType] {
     val id = "fihrist"
     val label = "Fihrist Authority"
   }
+
+  // http://estc.bl.uk/
+  case object ESTC extends IdentifierType {
+    val id = "bl-estc-citation-no"
```

Comment:
Why `citation`? I don't see it mentioned in the spec, they seem to call them [ESTC IDs, or estc identification number](https://estc.ucr.edu/starhelp/basic_search.html)? `bl-estc`? Or is this something more specific in the spec?

Reply:
This comes from Jonathan's initial issue; it does seem to used elsewhere: https://www.wikidata.org/wiki/Property:P3939

Reply:
Lovely, I was just wondering if it was referring to something specific. The WikiData article is more descriptive, and provbably useful to us, might be worth linking through to that.

Reply:
Added it to the comment.